code/stable/nopcm/Website/NoPCM_SRS.html

```diff
@@ -939,7 +939,7 @@ <h3>General Definitions</h3>
                                               </div> = q<sub>in</sub>&#8239;A<sub>in</sub>&minus;q<sub>out</sub>&#8239;A<sub>out</sub>&plus;g&#8239;V</em>
                 </div>
                 <p class="paragraph">
-                  Using the fact that <em>ρ</em>=<em>m</em>/<em>V</em>, (2) can be written as:
+                  Using the fact that <em>ρ</em>=<em>m</em>/<em>V</em>, Equation (2) can be written as:
```

Comment:
I know this is just a merge, so not the best place to comment -- but that hard-coded (2) is still a problem. And why did 'Equation' disappear?

Reply:
While merging, I noticed we had a hardcoded (2), so I used the `eqN` function, which is a known hack, so that it's easier to fix it later on. It was brought up somewhere else, but I don't remember where off the top of my head.